low-level/sensors/impl/generic/node_hw.py

```diff
@@ -1092,6 +1171,15 @@ def _parse_psu_unit_info(self, index, date, _time, sensor, sensor_num, event, st
 
         if is_last:
             specific_info.update(dynamic)
+        
+        if alert_type == "fault":
+            self.faulty_resources[sensor_id] = {
+                'status': status,
+                'event': event,
+                'resource': 'psu_unit'
```

Comment:
Resource name is already defined one. Please match with this.
```
    TYPE_PSU_UNIT = 'Power Unit'
```

Reply:
Done